packages/gatsby-source-graphql/package.json

```diff
@@ -29,7 +28,8 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "gatsby": "^2.0.15"
+    "gatsby": "^2.0.15",
+    "graphql": "^14.5.0"
```

Comment:
why move graphql to peerdependencies if you're not using it? Is it to satisfy other dependencies?

Reply:
oh yes you're right, we can get rid of it -- i'll open a new one though because i accidentally force-pushed...